src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2405,7 +2405,7 @@ class CgroupUtils:
                            "%s: requested:%s, assigned:%s" %
                            (socketlist, req_mem, available['memory']))
                 return {}
-            if mem not in assigned:
```

Comment:
@mike0042 I see that tmp_ngpus used on line 2393 is not defined anywhere in the python file. You may want to check that too.
It might be a good idea to run the whole file through pylint to find such similar problems.